sims/verilator/Makefile

```diff
@@ -1,112 +1,207 @@
-#########################################################################################
+#############################################################################
```

Comment:
Note: If we are deciding that 80 chars is good, then we should try to update the other files. Could we put an issue to track this.

Reply:
created the issue #564